js/client-mainmenu.js

```diff
@@ -314,21 +314,30 @@
 			var $chat = $pmWindow.find('.inner');
 			// this.tabComplete.reset();
 			this.chatHistories[userid].push(text);
-			if (text.toLowerCase() === '/ignore') {
+			var cmd = text;
```

Comment:
`= text.toLowerCase()` would save you some repetitive code.

Reply:
Well, it's not code that I had to repeat, but I see what you mean.